casper-application-domain/src/main/kotlin/hs/kr/entrydsm/domain/calculator/aggregates/Calculator.kt

```
@@ -0,0 +1,287 @@
+    )
+
+    /**
+     * 수식의 문법 유효성을 검사합니다.
+     *
+     * @param formula 검사할 수식
+     * @return 유효하면 true, 아니면 false
+     */
+    fun isValidFormula(formula: String): Boolean = try {
+        tokenize(formula)
+        true
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)